lib/Rex.pm

```diff
@@ -786,6 +786,12 @@ sub import {
         $found_feature = 1;
       }
 
+      if ( $add eq "task_chaining_cmdline_args" ) {
```

Comment:
It'd be nice if we could add a counterpart for this, like `no_task_chaining_cmdline_args`